src/language/lang_sw.php

```diff
@@ -0,0 +1,669 @@
+  'If you have any questions about what is or is not allowed, please ask an organizer.' =>
+    'Kama una maswali yoyote kuhusu kinacho ruhusiwa na kisicho ruhusiwa , tafadhali muulize muandaaji.',
+  'Have fun!' =>
+    'Kuwa na furaha!',
+  'Name' =>
+    'Jina',
+  'Email' =>
+    'Barua pepe',
+  'Token' =>
+    'Ishara(Token)',
+  'Team Registration' =>
+    'Usajili wa Timu',
+  'Team Name' =>
+    'Jina la Timu',
+  'Password' =>
+    'Neno Siri (Nywila)',
+  'Choose an Emblem' =>
+    'Chagua Nembo',
+  'Sign Up' =>
+    'Jiunge',
+  'Register to play Capture The Flag here. Once you have registered, you will be logged in.' =>
+    'Jisajili kucheza Capture the Flag hapa. Mara utakapo jisajili, utaingia ndani moja kwa moja.',
+  'Not Available' =>
+    'Haipo',
+  'Team Registration will be open soon, stay tuned!' =>
+    'Usajili wa timu utafunguliwa hivi karibuni, Kaa chonjo!',
+  'Try Again' =>
+    'Jaribu Tena',
+  'Select' =>
+    'Chagua',
+  'Team Login' =>
+    'Timu ya kuingia',
+  'Please login here. If you have not registered, you may do so by clicking "Sign Up" below. ' =>
+    'Tafadhali ingia hapa. Kama hujajisajili, unaweza kufanya hivyo kwa kubonyeza "Jiunge" Hapa chini. ',
+  'Team Login will be open soon, stay tuned!' =>
+    'Timu ya kuingia itakuwa wazi hivi karibuni, kaa chonjo!',
+  'ERROR' =>
+    'Makosa',
+  'Start Over' =>
+    'Anza upya',
+  'Window is too small' =>
+    'Dirisha ni dogo mno',
+  'For the best CTF experience, please make window size bigger.' =>
+    'Ili kufurahia vizuri CTF, tafadhali ongeza ukubwa wa dirisha.',
+  'Thank you.' =>
+    'Asante.',
+  'Logout' =>
+    'Ondoka',
+  'Registration' =>
+    'Usajili',
+  'Play CTF' =>
+    'Cheza CTF',
+  'Rules' =>
+    'Kanuni',
+  //Translations for GameboardController
+  'Admin' =>
+    'Kiongozi',
+  'ADMIN' =>
+    'Kiongozo',
+  'Navigation' =>
+    'Uelekezi',
+  'View Mode' =>
+    'Hali ya kuingia',
+  'View mode' =>
+    'Hali ya kuangalia',
+  'Tutorial' =>
+    'Mafunzo',
+  'Scoreboard' =>
+    'ubao (Ukurasa)',
+  'You' =>
+    'Wewe',
+  'Others' =>
+    'Wengine',
+  'All' =>
+    'Wote',
+  'Leaderboard' =>
+    'Ukurasa wa Kiongozi',
+  'Announcements' =>
+    'Matangazo',
+  'Teams' =>
+    'Timu',
+  'Filter' =>
+    'Vichujio',
+  'Activity' =>
+    'Kazi',
+  'Game Clock' =>
+    'Saa ya mchezo',
+  //Translations for AdminController
+  'Auto' =>
+    'Moja kwa moja',
+  'All Categories' =>
+    'Makundi yote',
+  'Open' =>
+    'Fungua',
+  'Tokenized' =>
+    'Isharisha',
+  'Hour' =>
+    'Saa',
+  'Hours' =>
+    'Masaa',
+  'Used by' =>
+    'Imetumiwa na',
+  'Used By' =>
+    'Imetumiwa na',
+  'Available' =>
+    'Inapatikana',
+  'Registration Tokens' =>
+    'Usajili wa Tokens',
+  'Create More' =>
+    'Tengeneza Zaidi',
+  'Export Available' =>
+    'Toa nje zilizopo',
```

Comment:
I suggest change to line 181 to be 'Uhamishaji Unapatikana'  instead of  'Uhamishaji haupatikani'